packages/system/cozystack-controller/Makefile

```
@@ -9,6 +9,7 @@ image: image-cozystack-controller update-version
 image-cozystack-controller:
 	docker buildx build -f images/cozystack-controller/Dockerfile ../../.. \
 		--provenance false \
+		--platform=$(PLATFORM) \
```

Let's make platform plural, since in the end there'll be at least `linux/amd64,linux/arm64` in this variable.
```suggestion
		--platform=$(PLATFORMS) \
```